dev/tests/integration/testsuite/Magento/Sales/Api/ShipmentTrackRepositoryInterfaceTest.php

```diff
@@ -41,14 +41,17 @@ public function testGetList()
         $filter3 = $filterBuilder->setField(ShipmentTrackInterface::TRACK_NUMBER)
             ->setValue('track number 4')
             ->create();
-        $filter4 = $filterBuilder->setField(ShipmentTrackInterface::CARRIER_CODE)
-            ->setValue('carrier code 5')
+        $filter4 = $filterBuilder->setField(ShipmentTrackInterface::TRACK_URL)
+            ->setValue('track url 5')
```

Comment:
It would be best to test multiple cases, which appear from this PR: without track url, with valid and invalid track url.